test/SparkLendMainnetIntegration.t.sol

```diff
@@ -226,64 +223,23 @@ contract SparkLendMainnetIntegrationTest is Test {
         
         RateTargetKinkInterestRateStrategy strategy
             = new RateTargetKinkInterestRateStrategy({
                 provider:                 poolAddressesProvider,
-                rateSource:               address(new PotRateSource(POT)),
+                rateSource:               address(new SSRRateSource(SUSDS)),
                 optimalUsageRatio:        0.95e27,
                 baseVariableBorrowRate:   0,
-                variableRateSlope1Spread: -0.004e27,  // -0.4% spread
+                variableRateSlope1Spread: 0.01e27,  // 1% spread
                 variableRateSlope2:       0.2e27
             });
-        IDefaultInterestRateStrategy prevStrategy
-            = IDefaultInterestRateStrategy(USDC_USDT_IRM);
-
-        assertEq(strategy.OPTIMAL_USAGE_RATIO(),       prevStrategy.OPTIMAL_USAGE_RATIO());
-        assertEq(strategy.MAX_EXCESS_USAGE_RATIO(),    prevStrategy.MAX_EXCESS_USAGE_RATIO());
-        assertEq(strategy.getBaseVariableBorrowRate(), prevStrategy.getBaseVariableBorrowRate());
-        assertEq(strategy.getVariableRateSlope1(),     prevStrategy.getVariableRateSlope1());
-        assertEq(strategy.getVariableRateSlope2(),     prevStrategy.getVariableRateSlope2());
-        assertEq(strategy.getMaxVariableBorrowRate(),  prevStrategy.getMaxVariableBorrowRate());
 
         _triggerUpdate(USDC);
         _triggerUpdate(USDT);
 
-        uint256 currentRateUSDC = 0.046649006348770788951303144e27;
-        uint256 currentRateUSDT = 0.035889274935287857936106840e27;
-
-        assertEq(_getBorrowRate(USDC), currentRateUSDC, "before: USDC mismatch");
-        assertEq(_getBorrowRate(USDT), currentRateUSDT, "before: USDT mismatch");
-
-        vm.startPrank(ADMIN);
-        configurator.setReserveInterestRateStrategyAddress(
-            USDC,
-            address(strategy)
-        );
-        configurator.setReserveInterestRateStrategyAddress(
-            USDT,
-            address(strategy)
-        );
-        vm.stopPrank();
-
-        _triggerUpdate(USDC);
-        _triggerUpdate(USDT);
-
-        // Should be no change
-        assertEq(_getBorrowRate(USDC), currentRateUSDC, "after1: USDC mismatch");
-        assertEq(_getBorrowRate(USDT), currentRateUSDT, "after1: USDT mismatch");
-
-        // Test with different parameters
-        strategy = new RateTargetKinkInterestRateStrategy({
-            provider:                 poolAddressesProvider,
-            rateSource:               address(new PotRateSource(POT)),
-            optimalUsageRatio:        0.95e27,
-            baseVariableBorrowRate:   0,
-            variableRateSlope1Spread: -0.002e27,  // -0.2% spread
-            variableRateSlope2:       0.2e27
-        });
+        assertEq(_getBorrowRate(USDC), 0.207489689165388369527167411e27, "before: USDC mismatch");
+        assertEq(_getBorrowRate(USDT), 0.057737691380706874411856558e27, "before: USDT mismatch");
```

Comment:
We should go back to this style use diffs cause its hard to interpret these large numbers otherwise
```solidity
uint256 currentRateUSDC = 0.046649006348770788951303144e27;
uint256 currentRateUSDT = 0.035889274935287857936106840e27;
```

Reply:
changed